src/sagemaker/clarify.py

```diff
@@ -1400,6 +1735,88 @@ def get_explainability_config(self):
         return copy.deepcopy({"shap": self.shap_config})
 
 
+class AsymmetricShapleyValueConfig(ExplainabilityConfig):
+    """Config class for Asymmetric Shapley value algorithm for time series explainability.
+
+    Asymmetric Shapley Values are a variant of the Shapley Value that drop the symmetry axiom [1].
+    We use these to determine how features contribute to the forecasting outcome. Asymmetric
+    Shapley values can take into account the temporal dependencies of the time series that
+    forecasting models take as input.
+
+    [1] Frye, Christopher, Colin Rowat, and Ilya Feige. "Asymmetric shapley values: incorporating
+    causal knowledge into model-agnostic explainability." NeurIPS (2020).
+    https://doi.org/10.48550/arXiv.1910.06358
+    """
+
+    def __init__(
+        self,
+        direction: Literal[
+            "chronological",
+            "anti_chronological",
+            "bidirectional",
+        ] = ASYM_SHAP_VAL_DEFAULT_EXPLANATION_DIRECTION,
+        granularity: Literal[
+            "timewise",
+            "fine_grained",
+        ] = ASYM_SHAP_VAL_DEFAULT_EXPLANATION_GRANULARITY,
+        num_samples: Optional[int] = None,
+        baseline: Optional[Union[str, Dict[str, Any]]] = None,
+    ):
+        """Initialises config for time series explainability with Asymmetric Shapley Values.
+
+        AsymmetricShapleyValueConfig is used specifically and only for TimeSeries explainability
+        purposes.
+
+        Args:
+            direction (str): Type of explanation to be used. Available explanation
+                types are ``"chronological"``, ``"anti_chronological"``, and ``"bidirectional"``.
+            granularity (str): Explanation granularity to be used. Available granularity options
+                are ``"timewise"`` and ``"fine_grained"``.
+            num_samples (None or int): Number of samples to be used in the Asymmetric Shapley
+                Value forecasting algorithm. Only applicable when using ``"fine_grained"``
+                explanations.
+            baseline (str or dict): Link to a baseline configuration or a dictionary for it.
```

Comment:
Do we support this for time-series explainability: `Link to a baseline configuration`? Did we test it?

Reply:
To my knowledge we support this but going to test this and will come back to you.

Reply:
We support this.